eslint-plugin/lib/rules/avoid-high-accuracy-geolocation.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (C) 2023 Green Code Initiative
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+"use strict";
+
+/** @type {import("eslint").Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: "suggestion",
+    docs: {
+      description: "Avoid using high accuracy geolocation in web applications.",
+      category: "eco-design",
+      recommended: "warn",
+    },
+    messages: {
+      AvoidUsingAccurateGeolocation:
+        "'{{notAllowedValue}}' is not a valid value, should be false.",
```

Comment:
Could you use a more precise message here?
It will be used in the SonarQube interface when a warning will be reported